### DISCUSSION TITLE
lipoprotein binding distribution

### DISCUSSION THREAD

**Discussion by Microbiotman** [Open-Systems-Pharmacology/Forum#1496] [Questions & Problems] [answered]
Hi all,

I would like to add lipoprotein binding in PK-Sim. But in the compound building block --> Basic physico-chemistry, there are only 3 choices : Albumin, alpha-acid glycoprotein or unknown.

So I have 2 questions : 
- If we choose unknown but we know and inform the fraction unbound, what does the software understand ?
- Is it possible to add lipoprotein binding in plasma with another way to proceed ?

Best,
Claire

**Comment by PavelBal** [comment, marked as answer]
> If we choose unknown but we know and inform the fraction unbound, what does the software understand ?

THe selection of albumin or alpha-acid glycoprotein as a binding partner has only an impact on the ontogeny of the binding partner. By selecting "unknow", no maturation of binding partner will be considered, but you still can model the fraction unbound.

> Is it possible to add lipoprotein binding in plasma with another way to proceed ?

You could add lipoprotein as a binding partner in the Expression-Building Block, set its expression to plasma, and implement binding of the compound to it. However, neither the free lipoprotein nor the drug-lipoprotein complex will be transported via blood flow. If you also want to consider blood flow transport of the complex, you would have to extend your model in MoBi.

**Reply by Microbiotman** [reply]
Thank you very much for your help. 
So I will try to build the model in MoBi ! ;)

**Reply by StephanSchaller** [reply]
Let us know about how it goes.We would be interested to see if this could then be integrated natively in PK-Sim.
Thanks! Stephan

**Reply by Microbiotman** [reply]
Hello everybody

I would like to share with you a little update about this project.

> You could add lipoprotein as a binding partner in the Expression-Building Block, set its expression to plasma, and implement binding of the compound to it.

That's what I did. However, I have a question. When PK-Sim gives the Compound concentration, is it the total concentration (free and complex with lipoprotein bindingpartner) or only the free compound concentration ? Indeed, when I run the newsimulation, I don't see any changing concerning compound concentration when I compare it with the model without lipoprotein bindingpartner added.

Thank you for your help ! :)

**Comment by StephanSchaller** [comment]
@Microbiotman , so there is no static binding concept similar to "Albumin" for lipoprotein binding in PK-Sim. 
If you use albumin as a surrogate, it might be an approximation, but given lipoprotein distribution is different to albumin, it will be inaccurate. Likely, uptake of bound lipoprotein complex to the liver is also higher and thus hepatic extraction would be underestimated.

Not sure someone in the community has looked at this in more detail but it would be a nice addition.
